docker/images/java-11-agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:11-alpine
+FROM eclipse-temurin:22-alpine
```

Comment:
This Java agent is supposed to run Java 11, does it not work on your architecture?